exercise1/exc1/broadcast/scripts/broadcaster.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import Int16
+
+
+def talker():
+    pub = rospy.Publisher('xamin', Int16, queue_size=10)
+    rospy.init_node('broadcaster', anonymous=True)
```

Comment:
```suggestion
    rospy.init_node('broadcaster', anonymous=True)
    pub = rospy.Publisher('xamin', Int16, queue_size=10)
```

I'm not sure if it makes a difference in this small example, but I would normally first initialize the node and then create the subscribers/publishers.